packages/integration-tests/src/process-controller/process_controller.ts

```diff
@@ -123,8 +123,16 @@ export const amplifyCli = (
 ): ProcessController => {
   let command: string;
   if (options?.installationType === 'local') {
```

Comment:
Could we get rid of this if statement and do the lookup unconditionally?

Reply:
easy to try.
If it works it works. if not we'll keep this.

Reply:
Tests are happy, we can do it.